### PR TITLE
functioning map which renders any array of groomers

### PR DESCRIPTION
**Pull request by hmerritt**
## Map
`GroomerMap` component which uses `Mapbox` to render the map.

Adds an icon layer to the map (used for each groomer) which renders a clickable icon for each groomer.
- custom callback function to access which groomer is selected

### Example Usage
```javascript
<GroomerMap groomers={groomers} onGroomerSelect={data => log("Selected groomer", data)} />
                    //^^^^^^^^ array of groomers
                                               //^^^^^^ callback function to access currently selected groomer data
```

## Misc
`env` helper variables
 - `isDevelopment`
 - `isProduction`

`log` function to only log during development
```javascript
log("my message")
```

### CODE REVIEW

**Review by josegocampo — DISMISSED** [review]
Simple amazing work Harry 👏👏👏

File: src/components/common/Mapbox.js [hunk 1]
```diff
@@ -149,6 +149,43 @@ const Mapbox = () => {
...
+        // handle queries with different capitalization than the source data by calling toLowerCase()
+        if (
+          feature.properties.businessName
+            .toLowerCase()
+            .search(query.toLowerCase()) !== -1
+        ) {
+          // add a tree emoji as a prefix for custom data results
+          // using carmen geojson format: https://github.com/mapbox/carmen/blob/master/carmen-geojson.md
+          feature['place_name'] = '🔎 ' + feature.properties.businessName;
+          feature['center'] = feature.geometry.coordinates;
```
Comment (sophiasagan) [inline src/components/common/Mapbox.js]: Are we keeping this emoji or the icon on the search bar?

File: src/styles/global.scss [hunk 2]
```diff
@@ -58,3 +55,104 @@ section {
...
+  position: relative;
+  border: 0;
+  border-radius: 0;
+  margin-top: 0;
+}
+
+.mapboxgl-ctrl-geocoder > div {
+  min-width: 100%;
+  margin-left: 0;
+}
```
Comment (sophiasagan) [inline src/styles/global.scss]: The styling looks great. I really like the blue.

File: src/components/mapbox/GroomerMap.js [hunk 3]
```diff
@@ -15,6 +19,10 @@ mapboxgl.accessToken = process.env.REACT_APP_MAPBOX_API || '';
 const GroomerMap = props => {
   // Mapbox element
   const $mapContainer = createRef();
+  const popUpRef = useRef(
+    new mapboxgl.Popup({ closeOnClick: false, offset: 15 })
+  );
+  const [map, setMap] = useState(null);
```
Comment (sophiasagan) [inline src/components/mapbox/GroomerMap.js:25]: You guys did a great job getting the popup issue sorted out

**Review by sophiasagan — APPROVED** [review]

**Review by sophiasagan — APPROVED** [review]

**Review by sophiasagan — APPROVED** [review]

**Review by josegocampo — APPROVED** [review]
I thought I had approved it already!

Good Job Harry!

**Pull request merged**